Formula/qt.rb

```diff
@@ -56,6 +54,14 @@ class Qt < Formula
   uses_from_macos "krb5"
   uses_from_macos "zlib"
 
+  on_macos do
+    if MacOS.version <= :mojave
+      depends_on xcode: [:build, :test]
+    else
+      depends_on xcode: [:build]
+    end
+  end
```

Comment:
We don't need to wrap this in an `on_macos` block, or test for `:mojave`. Just make it an unconditional
```ruby
depends_on xcode: :build
```
in the line you deleted above.

Reply:
> Before merging this, can we try to add the patch linked by @jonaski: https://codereview.qt-project.org/gitweb?p=qt/qtbase.git;a=patch;h=549a8f80a99280cf95631919c24a199415bef2d4? This sounds like a serious regression and the patch is small.

Ok, so we will need additional CI run. @carlocab what do you need about this patch?

Reply:
Looks fine.